api/src/test/java/org/openmrs/AllergenTest.java

```diff
@@ -11,14 +11,32 @@
 
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.Before;
 import org.openmrs.Concept;
+import org.openmrs.util.OpenmrsConstants;
+import org.openmrs.api.context.Context;
+import org.openmrs.test.BaseContextSensitiveTest;
 
 /**
  * Tests methods in {@link org.openmrs.Allergen}.
  */
-public class AllergenTest {
+public class AllergenTest extends BaseContextSensitiveTest {
+	
+	private static final String ALLERGY_OTHER_NONCODED_TEST_DATASET = "org/openmrs/api/include/otherNonCodedConcept.xml";
+	private String otherNonCodedConceptUuid;
```

Comment:
You can get rid of this local variable since you already setup Allergen.getOtherNonCodedConceptUuid()